src/ws/cockpit-wsinstance-https-factory@.service.in

```diff
@@ -1,10 +1,7 @@
 [Unit]
```

Comment:
Why is this still a template?

Reply:
The https-factory.socket still has `Accept=true`, which simplifies what the factory process has to do -- in particular, it can deal with one request at a time, do its thing, and die again. It can also skip the whole accept() stuff, but the biggest win is that it doesn't need to be a daemon which can serve arbitrarily many requests. This is quite important for this security sensitive bit.

Reply:
Ahh, and Accept=true needs the service to be a template.  Thanks!